lib/SILGen/SILGenApply.cpp

```diff
@@ -4558,7 +4558,7 @@ RValue SILGenFunction::emitApply(ResultPlanPtr &&resultPlan,
   }
 
   // Emit the raw application.
-  auto genericSig =
+  GenericSignature *genericSig =
     fn.getType().castTo<SILFunctionType>()->getGenericSignature();
```

Comment:
Ah, I see why you had the asymmetric `==` beffore. Yeah, this might be the best answer. Or changing the condition below to use `!`.

Reply:
Yeah, flipping the condition works too. Do you have a preference for either?

Reply:
Not really. Changing the condition seems slightly less likely to break if someone edits this code later, but not much.